src/utilities/filterFunctions.js

```diff
@@ -0,0 +1,37 @@
+export function sortFun(state, products) {
+    const sortedProducts = products.sort((a, b) => a["price"] - b["price"]);
```

Comment:
Like you have done for rating price Range, also use `Number()` explicit conversion here too

Reply:
Thanks for the Review, I have updated it ✅